File: Asp.Net.Angular.Template.Web/projects/clientApp/app/models/authentication-response.ts
import { Guid } from 'guid-typescript';

export interface IAuthenticationResponse {

  name: string;
  accessToken: string;
  email: string;
  userName: string;
  userAccountId: Guid;
}


export class AuthenticationResponse implements IAuthenticationResponse {
  public name: string;
  public accessToken: string;
  public email: string;
  public userName: string;
  public userAccountId: Guid;
}

File: Asp.Net.Angular.Template.Contract/Response/AuthenticationResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Asp.Net.Angular.Template.Contract.Response
{
    public class AuthenticationResponse
    {
        public string AccessToken { get; set; }
        public string Email { get; set; }
        public string UserName { get; set; }
        public Guid UserAccountId { get; set; }
    }
}
